### DISCUSSION TITLE
docs: EntityCollectionService or EntityServices

### DISCUSSION THREAD

**Discussion by pegaltier** [ngrx/platform#3096] [Q&A]
The current documentation contains 2 references to the same subject:
https://ngrx.io/guide/data/entity-collection-service
https://ngrx.io/guide/data/entity-services

## Other information:
We need to choose which one is the correct one and merge the useful information from the other one. Please help me for that and I will be able to create the PR.

## I would be willing to submit a PR for the docs :heart:

[X] Yes
[ ] No

**Comment by JiaLiPassion** [comment]
I think we should use `EntityCollectionService`.

**Comment by SerkanSipahi** [comment]
@pegaltier @JiaLiPassion  When im not wrong is `entity-collection-service` and `entity-services` are two different things (they do almost the same but they're a little different.):

entity-services (interface):
```ts
abstract class EntityServices {
  abstract entityActionErrors$: Observable<EntityAction>
  abstract entityCache$: Observable<EntityCache> | Store<EntityCache>
  abstract reducedActions$: Observable<Action>
  abstract dispatch(action: Action): void
  abstract getEntityCollectionService<T = any>(entityName: string): EntityCollectionService<T>
  abstract registerEntityCollectionService<T>(service: EntityCollectionService<T>): void
  abstract registerEntityCollectionServices(entityCollectionServices: EntityCollectionService<any>[]): void
}
```

entity-collection-service (interface):
```ts
class EntityCollectionServiceFactory {
  entityCollectionServiceElementsFactory: EntityCollectionServiceElementsFactory
  create<T, S$ extends EntitySelectors$<T> = EntitySelectors$<T>>(entityName: string): EntityCollectionService<T>
}
```

This is the only thing with the same behavior:
```ts
// entity-services
heroesService: EntityCollectionService<Hero> = entityServices.getEntityCollectionService('Hero');

// entity-collection-service
heroesService: EntityCollectionService<Hero> = EntityCollectionServiceFactory.create<Hero>('Hero');
```

It seems that `EntityServices` is more for advanced usage (maybe im wrong)!

**Comment by donohoea** [comment]
From the Docs:

**EntityServices**
EntityServices is a facade over the NgRx Data services and the NgRx Data EntityCache.

**Registry of EntityCollectionServices**
It is primarily a registry of EntityCollectionServices.

I do find the first line a little confusing but EntityServices and EntityCollectionServices are definitely to different things. As the docs say EntityServices is primarily a registry of EntityCollectionServices.
